doc/swagger.v1.yml

```diff
@@ -89,6 +89,42 @@ paths:
             $ref: '#/definitions/Error403'
       security:
         - foodsoft_auth: ['finance:user']
+    post:
+      summary: create new financial transaction
+      tags:
+        - 1. User
+        - 6. FinancialTransaction
+      parameters:
+        - in: body
+          name: body
+          description: financial transaction to create
+          required: true
+          schema:
+            $ref: '#/definitions/FinancialTransactionForCreate'
+      responses:
+        200:
+          description: success
+          schema:
+            type: object
+            properties:
+              financial_transaction:
+                $ref: '#/definitions/FinancialTransaction'
+        401:
+          description: not logged-in
+          schema:
+            $ref: '#/definitions/Error401'
+        403:
+          description: user has no ordergroup, is below minimum balance, or missing scope
+          schema:
+            $ref: '#/definitions/Error403'
+        404:
+          description: financial transaction type not found
+          schema:
+            $ref: '#/definitions/Error404'
+        422:
+          description: invalid parameter value
+          schema:
+            $ref: '#/definitions/Error422'
```

Comment:
It would be good to explain that this can only be used when `use_self_service` is enabled.
Perhaps in the endpoint's description, and also in the description of the accompanying return code (403).